### DISCUSSION TITLE
Can't get ignoreFile list work from Terraform

### DISCUSSION THREAD

**Discussion by gerbil** [aquasecurity/trivy-operator#1734] [Help & Support]
Terraform:

```hcl
trivy = {
  ignoreFile = "CVE-1970-0001 CVE-2023-49568"
}
```

```yaml
  {{- with .Values.trivy.ignoreFile }}
  trivy.ignoreFile: |
    {{- . | trim | nindent 4 }}
  {{- end }}
```

Resulting configmap:
```json
trivy.ignoreFile |
  CVE-1970-0001 CVE-2023-49568
```

This config is not working correctly since those CVEs are still visible.

**Comment by chen-keinan** [comment]
@gerbil I suggest to send a question in [aqua slack channel](https://aquasecurity.slack.com/join/shared_invite/zt-y1y94rja-nYw_BlF9meojdqr_8d4TUQ#/shared-invite/email) maybe someone from community has experience with it

**Reply by gerbil** [reply]
I will, thank you! 
But I can't see ignoreFIle anywhere in https://github.com/aquasecurity/trivy/tree/main/helm/trivy

**Comment by gerbil** [comment]
Possible wo:

```yaml
  {{- if .Values.trivy.ignoreFile }}
  trivy.ignoreFile: |
  {{- range .Values.trivy.ignoreFile }}
    {{ . | indent 4 }}
  {{- end }}
  {{- end }}
```

```hcl
  set_list {
    name  = "trivy.ignoreFile"
    value = ["CVE-1970-0001", "CVE-2023-49568"]
  }
```
